src/pages/Cart/CartInfoBox/CartInfoBox.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import InfoMapBox from './InfoMapBox';
+import './CartInfoBox.scss';
+
+export default function CartInfoBox({ infoBox }) {
+  const [sum, setSum] = useState(0);
+
+  const priceSum = () => {
+    let arr = [];
+    for (let i = 0; i < infoBox.length; i++) {
+      arr.push(infoBox[i].price);
+    }
+
+    let sum = arr.reduce((acc, cur) => acc + cur);
+
+    setSum(sum);
+  };
+
+  return (
+    <>
+      <div className="cartInfoBox">
+        <div className="allSelect">
+          <input type="checkbox" className="navSelect" onChange={priceSum} />
```

Comment:
```suggestion
          <input type="checkbox" className="navSelect" onClick={priceSum} />
```
체크박스 버튼을 클릭하는 이벤트이기 때문에, onClick이 어울려보입니다.

Reply:
적용완료했습니다~